src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderCharge.java

```diff
@@ -35,6 +38,32 @@
 @Table(name = "OFFENDER_CHARGES")
 @ToString(exclude = {"offenderBooking", "offenderCourtCase"})
 @With
+@NamedEntityGraph(
+    name = "charges-details",
+    attributeNodes = {
+        @NamedAttributeNode(value = "offenderCourtCase", subgraph = "court-case"),
+        @NamedAttributeNode(value = "offence"),
+        @NamedAttributeNode(value = "resultCodeOne"),
+        @NamedAttributeNode(value = "resultCodeTwo"),
```

Comment:
I didn't realise this needs doing for ManyToOnes as well. Surprised Hibernate doesn't do a join on them.

Reply:
Indeed, actually some of these joins were done by hibernate 6.1, but not 6.2